templates/monorepo/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "mono",
+  "private": true,
+  "devDependencies": {
+    "@types/react": "^16.9.43",
+    "@types/react-dom": "^16.9.8",
+    "lerna": "^3.15.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "tsdx": "^0.13.2",
+    "typescript": "^3.9.7"
+  },
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "lerna": "lerna",
+    "start": "lerna run start --stream --parallel",
```

Comment:
I've run into issues with the `--parallel` flag in the past. If component A relies on component B, you can't ensure that component B will be built first. Could that problem happen in this case? In my case, it took a long time to debug, and I wouldn't want consumers of this to get stuck on that. 

This does look like a slightly different use case, though, so maybe it's not a problem! Just thought it was worth the call out.